Model/Job.php

```diff
@@ -26,24 +32,24 @@ class Job extends \Magento\Framework\Model\AbstractModel implements JobInterface
 {
     protected $_eventPrefix = 'algoliasearch_queue_job';
 
-    /** @var \Magento\Framework\ObjectManagerInterface */
-    protected $objectManager;
+    /** @var ObjectManagerInterface */
```

Comment:
I like that we're steadily introducing stronger typing. With these changes I think the old PHPDoc becomes redundant and can be removed.